server/src/infra/repositories/asset.repository.ts

```diff
@@ -511,19 +511,35 @@ export class AssetRepository implements IAssetRepository {
           longitude: true,
         },
       },
-      where: {
-        ownerId: In([...ownerIds]),
-        isVisible: true,
-        isArchived,
-        exifInfo: {
-          latitude: Not(IsNull()),
-          longitude: Not(IsNull()),
+      where: [
```

Comment:
Is there a better way to write this OR expression?
`WHERE (ownerID IN ... OR albumId IN ...) AND isVisible = ... AND isArchived = ... AND ...`